src/components/bitcoin_theme_changer.js

```diff
@@ -1,9 +1,97 @@
 import React, { Component } from 'react';
 
 class BitcoinThemeChanger extends Component {
+
+  constructor(props){
+    super(props);
+
+    this.state = {
+      firstCondition: false,
+      secCondition: false,
+      thirdCondition: false,
+      fourthCondition: false,
+      fontSize: 24
+    }
+
+  }
+
+  _onClick(value){
+      switch (value) {
+                case 'green':
+                this.setState({
+                  firstCondition: !this.state.firstCondition,
+                  secCondition: false,
+                  thirdCondition: false,
+                  fourthCondition: false
+                })
+
+                break;
+                case 'blue':
+                  this.setState({
+                    firstCondition: false,
+                    secCondition: !this.state.secCondition,
+                    thirdCondition: false,
+                    fourthCondition: false
+                  });
+                break;
+                case 'yellow':
+                  this.setState({
+                    firstCondition: false,
+                    secCondition: false,
+                    thirdCondition: !this.state.thirdCondition,
+                    fourthCondition: false
+                  });
+                break;
+                case 'pink':
+                  this.setState({
+                    firstCondition: false,
+                    secCondition: false,
+                    thirdCondition: false,
+                    fourthCondition: !this.state.fourthCondition
+                  });
+                break;
+                default:
+                break;
+              }
+       this.props.getThemeColor(value);
+
+  };
+
+  addFontSize(value){
+    let newFontSize = this.state.fontSize + value;
+      this.setState({
+          fontSize: newFontSize
+      });
+      this.props.getAddFontSize(value);
```

Comment:
i need this, because i am getting the add font size value from themechanger component and passing it to index.js then passing it again to header component.since themechanger component is not directly communicating with header component. also on this instance i am trying the capabilities of sharing values using props and state without using ref method to directly call a function from another method.